libraries/chef_ingredient_resource.rb

```diff
@@ -41,6 +41,10 @@ class ChefIngredient < Chef::Resource::LWRPBase
 
       # Attribute to accept the license when applicable
       attribute :accept_license, kind_of: [TrueClass, FalseClass], default: false
+
+      # Attribute to enable selecting packages built for earlier versions in
+      # platforms that are not yet officially added to Chef support matrix
+      attribute :platform_version_backward_compat_mode, kind_of: [TrueClass, FalseClass], default: false
```

Comment:
It sounds like the group is moving toward consensus on `platform_version_compatability_mode` as the option name over in mixlib-install: https://github.com/chef/mixlib-install/pull/107#issuecomment-214858738

If that ends up being the case we should use the same attribute name here.